src/components/Search/SearchBar/SearchPopover.tsx

```diff
@@ -0,0 +1,119 @@
+import { useRef, useState } from "react";
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import IconButton from "@leafygreen-ui/icon-button";
+import { palette } from "@leafygreen-ui/palette";
+import Popover from "@leafygreen-ui/popover";
+import { Body, BodyProps } from "@leafygreen-ui/typography";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+import { useOnClickOutside } from "hooks";
+
+const { white, blue, gray } = palette;
+
+interface SearchPopoverProps {
+  disabled: boolean;
+  suggestions: string[];
+  onClick?: (suggestion: string) => void;
+}
+
+const SearchPopover: React.FC<SearchPopoverProps> = ({
+  disabled,
+  suggestions,
+  onClick,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const popoverRef = useRef<HTMLDivElement>(null);
+
+  useOnClickOutside([buttonRef, popoverRef], () => {
+    setIsOpen(false);
+  });
+
+  const handleClick = (suggestion: string) => {
+    setIsOpen(!isOpen);
+    onClick?.(suggestion);
+  };
+
+  return (
+    <SearchPopoverBox disabled={disabled}>
+      <IconButton
+        ref={buttonRef}
+        aria-labelledby="View search suggestions"
+        data-cy="search-suggestion-button"
+        disabled={disabled}
+        onClick={() => setIsOpen(!isOpen)}
+      >
+        <>
+          <Icon fill={gray.base} glyph="Bulb" />
+          <Icon fill={gray.base} glyph="CaretDown" />
+        </>
+        <Popover active={isOpen} data-cy="search-suggestion-popover">
+          <div ref={popoverRef}>
+            <StyledCard>
```

Comment:
How do you feel about adding a small title describing what these are similar to the one used in Compass. It would help establish the relationship to Project filters.

<img width="472" alt="image" src="https://github.com/evergreen-ci/parsley/assets/4605522/ed4b09be-048b-480e-ac01-f97b60b5e72e">